files/grest/rpc/script/reference_script_utxos.sql

```diff
@@ -0,0 +1,38 @@
+CREATE OR REPLACE FUNCTION grest.reference_script_utxos(_script_hashes text[])
+RETURNS TABLE (
+  script_hash text,
+  tx_hashes jsonb
+)
+LANGUAGE plpgsql
+AS $$
+DECLARE
+  _script_hashes_bytea  bytea[];
+BEGIN
+  -- convert input _script_hashes array into bytea array
+  SELECT INTO _script_hashes_bytea ARRAY_AGG(hashes_bytea)
+  FROM (
+    SELECT DECODE(hashes_hex, 'hex') AS hashes_bytea
+    FROM UNNEST(_script_hashes) AS hashes_hex
+  ) AS tmp;
+
+  RETURN QUERY (
+    SELECT
+      ENCODE(script.hash, 'hex'),
+      JSONB_AGG(
```

Comment:
While not really as relevant for this endpoint as some others - but for consistency , we started flattening output where possible (to allow for native filtering), would be preferable to use columns instead of JSON - as one can simply add create JSON from column on-the-fly using URL, but not the otherway round

Reply:
Actually, never mind, you can always filter that out from response as well if needed.